parquet-hadoop/src/main/java/org/apache/parquet/format/converter/ParquetMetadataConverter.java

```diff
@@ -81,6 +82,16 @@
 
   private static final Log LOG = Log.getLog(ParquetMetadataConverter.class);
 
+  private final boolean useSignedStringMinMax;
```

Comment:
Given that this controls all binaries except enum/unsigned int, can we rename this something like useSignedBinaryMinMax ?